doc/examples/sprite_batch/lib/main_auto_loading.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:math';
+
+import 'package:flame/sprite_batch.dart';
+import 'package:flame/components.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  final game = MyGame();
+  runApp(
+    GameWidget(
+      game: game,
+    ),
+  );
+}
+
+class MySpriteBatchComponent extends SpriteBatchComponent
```

Comment:
should we start putting new examples on the new app?

Reply:
We could, we still need to think on process for that though, like how could I write this example there, if the new feature is still on a PR?

Let's discuss more this after and we can start doing for the next PRs